app/views/observer/_form_observation_notes.html.erb

```diff
@@ -0,0 +1,70 @@
+<% # Notes section of create_observation form %>
+<div class="row push-down">
+  <%= content_tag(:span, "#{:NOTES.t} ", class: "bold") %>
+  <% if @observation.form_notes_parts(@user) == [Observation.other_notes_part]
+       part = Observation.other_notes_part %>
+
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 max-width-text">
+        <div>
+          <textarea class="form-control" rows="10"
+          id=<%= @observation.notes_part_id(part) %>
+          name=<%= @observation.notes_part_name(part) %>
+          ><%= @observation.notes_part_value(part) %></textarea>
```

Comment:
This ugly bit of html (the textarea lines 10-13) is because I want the the notes to appear in the params hash as params[:observation][:notes][:caption1] etc., rather than as params[:notes][:caption1]. 

Is there a better way to do this using Rails helpers?  I couldn't figure it out. (Maybe wrap_parameters? See http://api.rubyonrails.org/classes/ActionController/ParamsWrapper/ClassMethods.html#method-i-wrap_parameters.)